src/chrome/content/scaffold/scaffold.js

```diff
@@ -955,11 +955,10 @@ var Scaffold = new function() {
 	function updateSelectedTests() {
 		_clearOutput();
 		var listbox = document.getElementById("testing-listbox");
-		var items = listbox.selectedItems.slice();
+		var items = [...listbox.selectedItems];
 		if(!items || items.length == 0) return false; // No action if nothing selected
-		var i;
 		var tests = [];
-		for (i in items) {
+		for (var i=0; i<items.length; i++) {
 			items[i].getElementsByTagName("listcell")[1].setAttribute("label", "Updating");
```

Comment:
Using `listbox.selectedCount` above and for...of loop here on `listbox.selectedItems` would avoid the whole array intermediate. Spread operator and for...of use the same iterator, so it would be equivalent.